### PR TITLE
MANTA-5250 rust sharkspotter clap should check for option name instead of short id

### PR DESCRIPTION
**Pull request by rjloura**

### CODE REVIEW

File: src/config.rs [hunk 1]
```diff
@@ -153,4 +152,62 @@ impl Config {
 
         Ok(config)
     }
+
+    pub fn from_args(_args: std::env::Args) -> Result<Config, Error> {
```
Comment (papertigers) [inline src/config.rs:156]: What's the point of this if "_args" is never used?

Comment (rjloura) [reply src/config.rs]: Good question.  I think that was a holdover from when I first wrote this.  The clap examples did something like that.  Either way I removed it.

File: src/config.rs [hunk 2]
```diff
@@ -1,6 +1,6 @@
 // Copyright 2019 Joyent, Inc.
```
Comment (papertigers) [inline src/config.rs:1]: nit bump 2020

Comment (papertigers) [reply src/config.rs:1]: and in main.rs etc

Comment (rjloura) [reply src/config.rs]: Good catch!  Fixed them up.

**Pull request merged**